src/administrator/includes/rbacl.php

```diff
@@ -129,7 +129,9 @@ public static function check($userId, $client, $action, $contentId = null)
 					$userModel = self::model("user");
 					$contentRoleId = $userModel->getAssociatedContentRole($userId, $client, $contentId);
 
-					if (in_array($contentRoleId, $allowedRoles))
+					$rolesAllowed = array_intersect($contentRoleId, $allowedRoles);
```

Comment:
@sanjivani1812 Give some explanation why this is required

Reply:
In user model function getAssociatedContentRole - returns $db->loadColumn(); which return multiple data and here in_array checked which is not valid case